src/dispatch/entity/service.py

```diff
@@ -360,9 +364,18 @@ def _find_entities_by_regex_and_jsonpath_expression(
                                         entity_type=entity_type,
                                         project=signal_instance.project,
                                     )
-                except jsonpath_ng.JSONPathError:
-                    # field not found in signal_instance.raw
-                    pass
+                except KeyError:
+                    log.exception(
```

Comment:
Should these be logged as exceptions or info? With global entities wouldn't this fire quite a bit, even though it's large expected? Looking for a path that doesn't exist?

Reply:
I think it's an `exception`. If the key can't found be an empty list is returned.

Reply:
Eh, I'll downgrade it to `warning`. I do think it's a failure case.

Reply:
With the context, should it be `info`, or do you think `warning`?

Reply:
I think an exception would indicate something needs to be fixed/handled if we don't expect to take action a warning / info is probably fine imo.